contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol

```diff
@@ -97,7 +86,7 @@ abstract contract FunctionsBilling is HasRouter, IFunctionsBilling {
   // ================================================================
   // |                         Cost Events                          |
   // ================================================================
-  event BillingStart(bytes32 indexed requestId, Commitment commitment);
+  event BillingStart(bytes32 indexed requestId, IFunctionsRequest.Commitment commitment);
```

Comment:
Do we really need BillingStart/End events?
I don't see where it is useful, given that we have RequestStart/End carrying pretty much the same data.

Reply:
I think that we can remove them, but I'll do so in a separate PR

Reply:
If they're going then there's no need to extend them now.

Reply:
Sure. Usually it's preferred to have separation of concerns in PRs, but I'll remove them in this PR if you feel strongly about this line.

Reply:
I didn't realize you have to add "IFunctionsRequest" prefix. Disregard and keep it, sorry.